### ISSUE TITLE
invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by pogor** [etcinit/phabulous#63]
Hi! I have a problem. When I run the container I get the following errors:
```
DEBU[0008] epoch=1508505067&storyAuthorPHID=PHID-USER-rc4zdnym2dhfj7mri64q&storyData%5BobjectPHID%5D=PHID-CMIT-unyvog56oyceirr2nsmm&storyData%5BtransactionPHIDs%5D%5BPHID-XACT-CMIT-evokndi6l4iy2dp%5D=PHID-XACT-CMIT-evokndi6l4iy2dp&storyID=166298&storyText=belyshov+committed+rUFSD314156%3A+SAF-122+FIx+build.&storyType=PhabricatorApplicationTransactionFeedStory


2017/10/20 15:13:31 [Recovery] panic recovered:
POST /v1/feed/receive HTTP/1.1
Host: host:8085
Accept: */*
Content-Length: 350
Content-Type: application/x-www-form-urlencoded


runtime error: invalid memory address or nil pointer dereference
/usr/local/go/src/runtime/panic.go:489 (0x429eef)
        gopanic: reflectcall(nil, unsafe.Pointer(d.fn), deferArgs(d), uint32(d.siz), uint32(d.siz))
/usr/local/go/src/runtime/panic.go:63 (0x428d9e)
        panicmem: panic(memoryError)
/usr/local/go/src/runtime/signal_unix.go:290 (0x43ed2f)
        sigpanic: panicmem()
/go/src/github.com/etcinit/phabulous/app/controllers/feed.go:56 (0x84b6b1)
        (*FeedController).postReceive: if res.URI != "" {
/go/src/github.com/etcinit/phabulous/app/controllers/feed.go:30 (0x84ce44)
        postReceive)-fm: front.POST("/receive", f.postReceive)
/go/src/github.com/gin-gonic/gin/context.go:97 (0x83995a)
        (*Context).Next: c.handlers[c.index](c)
/go/src/github.com/gin-gonic/gin/recovery.go:45 (0x848daa)
        RecoveryWithWriter.func1: c.Next()
/go/src/github.com/gin-gonic/gin/context.go:97 (0x83995a)
        (*Context).Next: c.handlers[c.index](c)
/go/src/github.com/gin-gonic/gin/logger.go:72 (0x84807f)
        LoggerWithWriter.func1: c.Next()
/go/src/github.com/gin-gonic/gin/context.go:97 (0x83995a)
        (*Context).Next: c.handlers[c.index](c)
/go/src/github.com/gin-gonic/gin/gin.go:284 (0x83fba0)
        (*Engine).handleHTTPRequest: context.Next()
/go/src/github.com/gin-gonic/gin/gin.go:265 (0x83f4bb)
        (*Engine).ServeHTTP: engine.handleHTTPRequest(c)
/usr/local/go/src/net/http/server.go:2568 (0x6ccf62)
        serverHandler.ServeHTTP: handler.ServeHTTP(rw, req)
/usr/local/go/src/net/http/server.go:1825 (0x6c91d2)
        (*conn).serve: serverHandler{c.server}.ServeHTTP(w, w.req)
/usr/local/go/src/runtime/asm_amd64.s:2197 (0x455dd1)
        goexit: BYTE    $0x90   // NOP
```

**Comment by turboza** [comment]
This may duplicate with #39

**Comment by schemar** [comment]
Regarding this ticket as closed, I created #67 instead.
